WordPress/src/main/res/values/strings.xml

```diff
@@ -2224,18 +2224,18 @@
     <string name="reader_empty_posts_no_connection" translatable="false">@string/no_network_title</string>
     <string name="reader_empty_posts_request_failed">Unable to retrieve posts</string>
     <string name="reader_self_hosted_select_filter">Use the filter button to find posts on specific subjects</string>
-    <string name="reader_empty_posts_in_tag">No posts with this topic</string>
+    <string name="reader_no_posts_with_this_tag">No posts with this tag</string>
     <string name="reader_empty_posts_in_tag_updating">Fetching posts…</string>
-    <string name="reader_empty_posts_in_custom_list">The sites in this list haven\'t posted anything recently</string>
-    <string name="reader_empty_followed_tags_subtitle">Add topics here to find posts about your favorite topics</string>
-    <string name="reader_empty_followed_tags_title">No followed topics</string>
-    <string name="reader_empty_recommended_blogs">No recommended sites</string>
-    <string name="reader_empty_followed_blogs_title">No followed sites</string>
-    <string name="reader_empty_followed_blogs_search_title">No sites matching your search</string>
-    <string name="reader_empty_followed_blogs_description">When you follow sites, you\'ll see their content here</string>
+    <string name="reader_empty_blogs_posts_in_custom_list">The blogs in this list have not posted anything recently</string>
+    <string name="reader_empty_subscribed_tags_subtitle">Add tags here to find posts about your favorite tags</string>
```

Comment:
💡 I think the last "topics" in this string was meant as the actual meaning of an interest topic, so the sentence doesn't make much sense now. I would suggest keeping the word "topics" there or maybe change to "interests", wdyt?

vorite tags
```suggestion
    <string name="reader_empty_subscribed_tags_subtitle">Add tags here to find posts about your favorite topics</string>
```

Reply:
Makes sense 👍